_posts/2015-12-16-new-office-ashford.html

```diff
@@ -0,0 +1,18 @@
+---
+layout: 		post
+title:  		"Check out our Space between RHL and some other firm"
+date:   		2015-12-16
+author: 		"Marcus Cooke"
+authorPhoto:	"2aa226fd35a96b7dcf125f686d944245"
+categories: 	space between
+banner:			"http://www.gratisography.com/pictures/213_1.jpg"
+---
+
+Since day 1 we have been graciously hosted by HR GO. But as new team we need a place to build a business together. Part of that is being able to have the conversions you need have in a space where everyone feels comfortable saying what they need to say. We need to be able to bring clients to our space and have discussions, meetings and presentations. We can now proudly say we have that space.
+
+It started out as a spare, furniture-packed, room belonging to RHL, just across the road from HR GO. We were graciously given the pick of furniture we wanted from the room. The room was full of the discarded love children from Ikea-Staples. We chose the best suited pieces and then moved the rest into another room that was to become a graveyard for the sad and unslightly children.
```

Comment:
You used graciously twice in this post, I would change to just luckily or something as it stands out